.semaphore/semaphore.yml

```diff
@@ -105,7 +105,7 @@ blocks:
 
 - name: "Typha"
   run:
-    when: "change_in('/api/') or change_in('/libcalico-go/') or change_in('/typha/')"
+    when: "change_in(['/*', '/api/', '/libcalico-go/', '/typha/'], {exclude: ['/.semaphore', '/**/README.md', '/**/LICENSE']})"
```

Comment:
I don't think excluding ".semaphore" is going to be useful, because we probably want to trigger tests when semaphore config changes.

Reply:
Yeah - I agree - but just testing some stuff at the mo and didn't want to be triggering everything :-)

Reply:
t'is removed.